src/components/MyMap/MyMap.js

```diff
@@ -1,44 +1,27 @@
-import React, { useState, useCallback } from 'react';
-import { GoogleMap, LoadScript } from '@react-google-maps/api';
+import React, { Component } from 'react';
+import { Map, GoogleApiWrapper } from 'google-maps-react';
 
-const containerStyle = {
-  width: '400px',
-  height: '400px',
+const mapStyles = {
```

Comment:
Did you mean to use this or can we remove it?

Reply:
deleted